addons/sale/i18n/sale.pot

```diff
@@ -3579,9 +3591,11 @@ msgid "The order is not in a state requiring customer signature."
 msgstr ""
 
 #. module: sale
-#: code:addons/sale/models/payment.py:97
```

Comment:
This change is not needed: line numbers do not matter (they change because of fixes and forward-port, it's fine if it doesn't match). Not sure why you changed the string below.

Reply:
pot is generated in odoo

Reply:
Since 13.0, the line number in exported .pot are set to 0 to avoid useless diff at each reexport... but it generates a useless diff the first time...